### DISCUSSION TITLE
Even the simple config doesn't seem to work for me

### DISCUSSION THREAD

**Discussion by GRX** [dswd/vpncloud#265] [Q&A]
Hi!

I've been trying to get a Docker swarm working on a Unifi site-to-site VPN but ran into that makes me want to try vpncloud (https://serverfault.com/questions/1085063/docker-swarm-over-ipsec-vti-unifi-site-to-site)

I have little know of what I'm doing to be honest :)
First I installed vpncloud on server 1 which is at home.
Second I installed vpncloud on server 2 which is remote
Third I installed vpncloud on server 3 which is on the same remote as 2

Before configuring the 3th server, I thought to test between  server 1 and server 2.
On server 1 I ran:
`sudo vpncloud --password 'mylittlesecret' --ip 10.0.0.1 -c {PUBLIC_IP_OF_REMOTE}:1000` (port forwarded to correct machine)

On server 2 I ran:
`sudo vpncloud --password 'mylittlesecret' --ip 10.0.0.2 -c {PUBLIC_IP_OF_HOME}:3000` (port forwarded to correct machine)

Then vpncloud start happily on both machines, but I can't ping one another;
On server 1 I can ping `10.0.01` but not `10.0.0.2`.
On server 2 I can ping `10.0.0.2` but not `10.0.0.1`.

I'm super beginner in this whole world of networking and VPN so please bear with me if I miss something obvious.

Thank you in advance!

**Comment by dswd** [comment]
Hi @GRX,

sorry for the late reply. I don't have much time to support users right now.
Please check the output of the commands to whether they actually connected or not. I suspect that something with your port mapping in your NAT router is wrong. You could specify a listen port using `-l` to make sure you forward to the correct port on the machines.
